packages/components/src/theme/ThemeUtils.test.ts

```diff
@@ -333,6 +336,47 @@ describe('getDefaultBaseThemes', () => {
       },
     ]);
   });
+
+  /**
+   * This test is to ensure that the css variables in theme-dark and theme-light are in sync,
+   * to prevent the case where a css variable is added to one theme and not the other.
+   */
+  it('light and dark themes should contain the same css variables', () => {
+    const themeDarkDir = './packages/components/src/theme/theme-dark';
+    const themeLightDir = './packages/components/src/theme/theme-light';
+
+    function extractCssVariablesFromFiles(directory: string): Set<string>[] {
+      const files = fs.readdirSync(directory);
+      const fileVariables = [] as Set<string>[];
+      files.forEach(file => {
+        const variables = new Set<string>();
+        const filePath = path.join(directory, file);
+        const fileContent = fs.readFileSync(filePath, 'utf8');
+        // strip comments as they may contain css variable names
+        const fileContentsNoComments = fileContent.replace(
+          /\/\*[\s\S]*?\*\//g,
+          ''
+        );
+        const lines = fileContentsNoComments.split('\n');
+        // extract css variables from each line, before the colon
+        const matches = lines.map(line => line.match(/(--[a-zA-Z-]+):/)?.[1]);
+        matches?.forEach(match => {
+          if (match != null) return variables.add(match as string);
+        });
+        fileVariables.push(new Set([...variables].sort()));
```

Comment:
I'm not sure the sorting is necessary here unless I'm misreading. I think jest will treat this as true
```
expect(new Set('ABC')).toEqual(new Set('BCA'));
```

Reply:
It's not necessary for the comparison no. But it seemed to make the results more predicatble for humans.